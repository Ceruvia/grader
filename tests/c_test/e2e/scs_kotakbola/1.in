10 5 2
H 10
M 2
H 5
M 3
E