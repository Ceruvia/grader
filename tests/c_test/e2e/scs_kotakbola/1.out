15 5 0
Tidak
